src/query/api/v1/handler/placement/delete_all.go

```diff
@@ -33,8 +33,12 @@ import (
 )
 
 const (
-	// DeleteAllURL is the url for the handler to delete all placements (with the DELETE method).
-	DeleteAllURL = handler.RoutePrefixV1 + "/placement"
+	// OldM3DBDeleteAllURL is the old url for the handler to delete all placements, maintained
+	// for backwards compatibility.
+	OldM3DBDeleteAllURL = handler.RoutePrefixV1 + "/placement"
```

Comment:
hmm, this does not contain `delete` in it?

Reply:
nope, just DELETE method